benchmarks/bench_packed_kda_decode.py

```
@@ -0,0 +1,315 @@
+        raise RuntimeError("packed-input CuTe KDA decode requires exact CC 10.0")
+
+    modes = ("direct", "cuda_graph") if args.mode == "both" else (args.mode,)
+    rows = []
+    print(f"{'mode':<10} {'B':>5} {'tile':>6} {'median_us':>12} {'logical_TB/s':>14}")
+    for ordinal, batch in enumerate(args.batch_size):
+        case = _make_case(batch, device, args.seed + ordinal)
+        tile_v = args.tile_v or _select_tile_v(batch)
+        output_error, state_error = _check(case, tile_v)
+        for mode in modes:
+            run = (
+                (lambda: _run(case, tile_v))
```

Ruff (0.16.1)

[warning] Function definition does not bind loop variable `case`

(B023)